### ISSUE TITLE
2.0.8 fails horology tests on 32-bit i386 Debian

### ISSUE THREAD

**Issue by df7cb** [pllua/pllua#8]
2.0.8 fails the horology tests on 32-bit i386 Debian (all distribution releases, all PG versions). amd64/arm64/ppc64el are fine.

```
16:26:15 --- /<<PKGBUILDDIR>>/expected/horology.out	2021-02-14 21:00:41.000000000 +0000
16:26:15 +++ /<<PKGBUILDDIR>>/results/horology.out	2021-04-22 14:26:14.041554324 +0000
16:26:15 @@ -40,7 +40,7 @@
16:26:15  $$;
16:26:15  INFO:  1970-01-01 00:00:00+00
16:26:15  INFO:  2019-04-22 00:00:00+00
16:26:15 -INFO:  2019-04-22 00:00:00.000001+00
16:26:15 +INFO:  1969-12-31 23:24:12.516352+00
16:26:15  INFO:  2019-04-22 00:00:00.001+00
...
```
https://pgdgbuild.dus.dg-i.net/job/postgresql-pllua-binaries/architecture=i386,distribution=buster/46/console
https://pgdgbuild.dus.dg-i.net/job/postgresql-pllua-binaries/46/

**Comment by df7cb** [comment]
This regression diff chunk seems to hint at an integer truncation problem:
```
16:26:08 -INFO:  epoch_usec	-51916498765433
16:26:08 +INFO:  epoch_usec	-2147483648
```

**Comment by RhodiumToad** [comment]
Should be fixed in 2.0.9 (just pushed).
